prometheus/openshift/files/params/metrics-params

```diff
@@ -0,0 +1,6 @@
+NAMESPACE=monitoring
+GRAFANA_DATA_VOL_SIZE=20Gi
+HAPROXY_STATS_PASSWORD=65IQD9X1IF
+USER=prometheus
+GRAFANA_PASSWORD=secret
+RC_WEBHOOK='myRocketChatWebHook'
```

Comment:
What happens if the target environment does not want to integrate with rocketchat? I.e.: is this optional? It should be.

Reply:
Yes, this is still optional